### DISCUSSION TITLE
exit status 1: The connection to the server 0.0.0.0:49624 was refused - did you specify the right host or port?

### DISCUSSION THREAD

**Discussion by jyotishmansarma** [egovernments/Digit-Core#260] [DIGIT Platform]
Hi,
after executing sudo go run digit_setup.go , getting following errors
2023/10/07 16:19:19 Helm Directory - /Users/jyotishmansarma/Documents/project/DIGIT-DevOps/deploy-as-code/helm
2023/10/07 16:19:19 ------------------------------------ DEPLOYING CLUSTER CONFIGS ------------------------------------
/Users/jyotishmansarma/Documents/project/DIGIT-DevOps/deploy-as-code/helm/charts/cluster-configs
2023/10/07 16:19:19 helm template -f /Users/jyotishmansarma/Documents/project/DIGIT-DevOps/deploy-as-code/helm/environments/quickstart-config.yaml --output-dir /tmp/helm-626580554 -f /Users/jyotishmansarma/Documents/project/DIGIT-DevOps/deploy-as-code/helm/environments/quickstart-config-secrets.yaml .
2023/10/07 16:19:20 exit status 1: The connection to the server 0.0.0.0:49624 was refused - did you specify the right host or port?

panic: exit status 1: The connection to the server 0.0.0.0:49624 was refused - did you specify the right host or port?

**Comment by nikhilmulinti** [comment]
Hi @jyotishmansarma Please make sure kubeconfig is exported before you run the go script, you can verify by running kubectl get nodes, see if you are able to connect to cluster

**Reply by jyotishmansarma** [reply]
Hi @nikhilmulinti-egov we have resolved this issues but now we are getting following issues.
![WhatsApp Image 2023-10-11 at 2 57 38 PM](https://github.com/egovernments/Digit-Core/assets/65509586/c96432a7-dff8-435c-babf-82d21150fb1d)

**Reply by nikhilmulinti** [reply]
Hi @jyotishmansarma  We are not supporting quick installation setup.For better experience of DIGIT, please make use of this document to install it on AWS.
https://core.digit.org/guides/installation-guide/quick-setup/digit-installation-on-aws

**Reply by Ros-web** [reply]
Hi @jyotishmansarma - Got any solution for  CrashLoopBackOff, i am facing same issues also.
![image](https://github.com/egovernments/Digit-Core/assets/60926594/eb8b8a05-efd5-464d-9a6e-e4a04701d5a3)
